src/Sarif.Driver/Sdk/CommonOptionsBase.cs

```diff
@@ -12,9 +12,21 @@ public class CommonOptionsBase
             'p',
             "pretty-print",
             Default = false,
-            HelpText = "Produce pretty-printed JSON output rather than compact form.")]
+            HelpText =
+            "Produce pretty-printed JSON output rather than compact output (all white space removed). If neither " +
+            "--pretty-print nor --minify is specified, --pretty-print is set to true. --pretty-print and --minify " +
+            "cannot be specified together.")]
         public bool PrettyPrint { get; set; }
 
+        [Option(
+            'm',
+            "minify",
+            Default = false,
+            HelpText = "Produce compact JSON output (all white space removed) rather than pretty-printed output. " +
+            "If neither --pretty-print nor --minify is specified, --pretty-print is set to true. --pretty-print " +
+            "and --minify cannot be specified together.")]
+        public bool Minify { get; set; }
```

Comment:
>Minify [](start = 20, length = 6)

Hey, did we describe this change in the release notes yet?

Reply:
Oops

---
In reply to: [500521496](https://github.com/microsoft/sarif-sdk/pull/2100#discussion_r500521496) [](ancestors = 500521496)

Reply:
Updated.

---
In reply to: [500522977](https://github.com/microsoft/sarif-sdk/pull/2100#discussion_r500522977) [](ancestors = 500522977,500521496)